README.md

```diff
@@ -25,6 +25,10 @@ Indicate whether the user has agreed to that category of cookies. The
 only category at the moment is `analytics` (and `essential`, but that
 will always return true).
 
+#### `GovWifi.cookies.setCategoryAllowed(categoryName, isAllowed)`
+Amend the cookie policy to allow or disable the given
+`categoryName`. If `categoryName` is not recognised do nothing.
```

Comment:
I don't know much about frontend stuff; is silent failure like this generally considered the way to go in a frontend context?

Either way, good that the README specifically says what to expect in that scenario.

Reply:
For non-critical operations I would say yes. Server-side I'd probably crash but here it seems fine to just happily ignore any cookie-writing that we don't recognise.